ios/RCTVideo.m

```diff
@@ -330,7 +333,10 @@ - (AVPlayerItem*)playerItemForSource:(NSDictionary *)source
   if (isNetwork) {
     NSArray *cookies = [[NSHTTPCookieStorage sharedHTTPCookieStorage] cookies];
     AVURLAsset *asset = [AVURLAsset URLAssetWithURL:url options:@{AVURLAssetHTTPCookiesKey : cookies,
-                                                                  @"AVURLAssetHTTPHeaderFieldsKey": @{@"X-JWT-TOKEN": oauthToken}}];
+                                                                  @"AVURLAssetHTTPHeaderFieldsKey": @{oauthTokenKey: oauthToken}}];
```

Comment:
AFAIK it used private API it will cause rejection, also there is no point adding resource loader delegate if we're keepign this ;)